### PR TITLE
Four of the first 5

### PR DESCRIPTION
**Pull request by rgee0**
Hello Alex,

4 out of 5 to review  - I was intending to also commit rgb in this first batch.  

I'm now intending to change rgb to incorporate flags and likely rework cheerlights to function in a similar way to the space example.

### CODE REVIEW

**Comment by alexellis** [comment]
Nice, do you want me to wait and review then?

**Comment by rgee0** [comment]
Review was more of a word choice than expectation.

Waiting paid off tho as my waking thought this morning was that I hadn't changed the source file names. Also changed the go_blinkt typo in instructions in the top level readme

File: random_blink/app.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	for i := len(arr) - 1; i > 0; i-- {
+		j := rand.Intn(i)
+		arr[i], arr[j] = arr[j], arr[i]
+	}
+
+	subsetSize := rand.Intn(5) + 1 // +1 as zero based
+	return arr[:subsetSize]
+}
+
+func IsIn(s *[]int, e *int) bool {
```
Comment (alexellis) [inline random_blink/app.go:31]: Just a note, Golang controls visibility of functions through casing. So lowercase 1st letter is like private/internal and upper first letter is public, as in exported and available to other packages.

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Running one of the examples:
 ```
 export GOPATH=$HOME/go/
 mkdir -p /home/pi/go/src/github.com/alexellis/
-git clone https://github.com/alexellis/go_blinkt_examples
+git clone https://github.com/alexellis/blinkt_go_examples
```
Comment (alexellis) [inline README.md:12]: Good spot!

**Comment by alexellis** [comment]
Merged (check in Master) Thanks for the changes.

**Pull request closed without merging**